pygeos/constructive.py

```diff
@@ -486,6 +487,32 @@ def point_on_surface(geometry, **kwargs):
     return lib.point_on_surface(geometry, **kwargs)
 
 
+def polygonize(geometries, **kwargs):
+    """Creates polygons formed from the linework of a set of Geometries.
+
+    Polygonizes a set of Geometries which contain linework that
```

Comment:
```suggestion
    Polygonizes an array of Geometries that contain linework which
```

A small that-vs-which bikeshedding here. Also: `set` is confusing in the context of python.